src/tools/rbd_mirror/PoolReplayer.cc

```diff
@@ -422,10 +422,9 @@ int PoolReplayer<I>::init_rados(const std::string &cluster_name,
   // librados::Rados::conf_read_file
   int r = cct->_conf->parse_config_files(nullptr, nullptr, 0);
   if (r < 0) {
```

Comment:
@dillaman In master, this line says `if (r < 0 && r != -ENOENT) {` (as a consequence of a7fd06637a9242ed6609f48b78408dca7c2e5b8a which is not being backported (?))

Reply:
That's fine -- we don't want to backport that other PR since that's a new feature. I would just keep the master/nautilus `if (r < 0 && r != -ENOENT)` check

Reply:
@dillaman Hm. I'm confused. In master, we have:

```
  // librados::Rados::conf_read_file
  int r = cct->_conf.parse_config_files(nullptr, nullptr, 0); 
  if (r < 0 && r != -ENOENT) {
    // do not treat this as fatal, it might still be able to connect
    derr << "could not read ceph conf for " << description << ": "
         << cpp_strerror(r) << dendl;
  }
```

But shouldn't it be

```
if (r < 0) {
  if (r == -ENOENT) {
    cct->put();
    return r;
  }
  // do not treat this as fatal, it might still be able to connect
  derr << "could not read ceph conf for " << description << ": "
       << cpp_strerror(r) << dendl;
}
```

Reply:
No, we don't want to emit a message on `-ENOENT`, so the new code would be ...

```
  if (r < 0 && r != -ENOENT) {
    // do not treat this as fatal, it might still be able to connect
    derr << "could not read ceph conf for " << description << ": "
         << cpp_strerror(r) << dendl;
  }
```

Reply:
@dillaman ACK